projects/1.20-fabric/assets/xp-storage/xp_storage/lang/zh_cn.json

```
@@ -0,0 +1,38 @@
+{
+  "item.xp_storage.crystallized_lapis": "晶化青金石",
+  "item.xp_storage.xp_book_upgrade_smithing_template": "锻造模板",
+  "item.xp_storage.smithing_template.xp_book_upgrade.additions_slot_description": "Add钻石、下界合金锭或下界之星",
+  "item.xp_storage.smithing_template.xp_book_upgrade.applies_to": "经验之书",
+  "item.xp_storage.smithing_template.xp_book_upgrade.base_slot_description": "Add一本经验之书",
+  "item.xp_storage.smithing_template.xp_book_upgrade.ingredients": "钻石、下界合金锭或下界之星",
+  "item.xp_storage.xp_book": "经验之书",
+  "item.xp_storage.xp_book2": "二级经验之书",
+  "item.xp_storage.xp_book3": "三级经验之书",
+  "item.xp_storage.tooltip": "存储的经验值：",
+  "item.xp_storage.xp_book.tooltip.upgrade": "升级： %1$s",
+  "item.xp_storage.xp_book.tooltip.level": "%1$s / %2$s级",
+  "item.xp_storage.xp_book.tooltip.point": "%1$s / %2$s点",
+  "item.xp_storage.xp_bottle.tooltip.level": "%1$s级",
+  "item.xp_storage.xp_bottle.tooltip.point": "%1$s点",
+  "item.xp_storage.tooltip.deprecated": "需要升级！",
+  "item.xp_storage.tooltip.deprecated_how": "放入工作台",
+
+  "upgrade.xp_storage.xp_book_upgrade": "经验之书升级",
+
+  "text.autoconfig.xp_storage.title": "经验存储配置",
+
+  "text.autoconfig.xp_storage.category.books": "书",
+  "text.autoconfig.xp_storage.option.books.capacity": "容量（lvl）",
+  "text.autoconfig.xp_storage.option.books.xpFromUsing": "使用时获取的经验值（%）",
+  "text.autoconfig.xp_storage.option.books.xpFromBrewing": "每瓶增加的等级",
+  "text.autoconfig.xp_storage.option.books.barColor": "条状栏颜色",
+
+  "text.autoconfig.xp_storage.category.bottles": "瓶",
+  "text.autoconfig.xp_storage.option.bottles.enableBrewing": "开启酿造经验",
+  "text.autoconfig.xp_storage.option.bottles.maxLevel": "容量（lvl）",
+
+  "text.autoconfig.xp_storage.category.cosmetic": "外观",

```

```suggestion
  "text.autoconfig.xp_storage.category.cosmetic": "装饰",

```
根据下面内容来看，这似乎是辅助性的功能，并不涉及改编物品外观吧？